.github/workflows/static.yml

```diff
@@ -46,6 +46,6 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Psalm
-        uses: docker://vimeo/psalm-github-actions:3.14.2
+        uses: docker://vimeo/psalm-github-actions:3.16
```

Comment:
I just added the feature to tag the docker image. 
It will automatically tag all new versions after 3.4.2. But “after” I mean by date and not version number. 

It will automatically tag a new docket tag on next Packagist version.

Reply:
shall we switch to another image?

Reply:
We can just wait a while. Next release of psalm will be properly tagged. Im waiting for 3.16.1

(It has not been any merged in psalm for 4 weeks)

Reply:
* [x] new version released
* [x] Docker image built and deployed
* [x] ci green